definitions/json/AuthorisationCaseField.json

```diff
@@ -5993,6 +5993,762 @@
     "UserRole": "caseworker-employment-api",
     "CRUD": "CRUD"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "existingJurisdictionCodes",
+    "UserRole": "caseworker-employment-etjudge",
```

Comment:
Change UseRole access for caseworker-employment-etjudge and caseworker-employment to R for all fields.